src/lens/EphemeralGetPopulatedTicksInRange.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "./TickLens.sol";
+
+/// @notice A lens that fetches chunks of tick data in a range for a Uniswap v3 pool without deployment
+/// @author Aperture Finance
+/// @dev The return data can be accessed externally by `eth_call` without a `to` address or internally by catching the
+/// revert data, and decoded by `abi.decode(data, (PopulatedTick[]))`
+contract EphemeralGetPopulatedTicksInRange is TickLens {
```

Comment:
One proposal: merge the constructors from `EphemeralGetPopulatedTicksInRange` and `EphemeralGetPopulatedTicksInWord` to `TickLens`? That way, we can have a unified constructor and we can pass in argument to the unified constructor to specify which query we want to run.

Reply:
NVM. We can do this for newer queries. Don't want you to spend too much time on refactoring.